lib/Requestable.js

```diff
@@ -256,9 +256,17 @@ class Requestable {
             results.push(...thisGroup);
 
             const nextUrl = getNextPage(response.headers.link);
-            if (nextUrl && !(options && typeof options.page !== 'number')) {
-               log(`getting next page: ${nextUrl}`);
-               return this._requestAllPages(nextUrl, options, cb, results);
+            if(nextUrl) {
+               options.page = parseInt(
+                 nextUrl.match(/(&page=[0-9]*)/g)
```

Comment:
This only matches against the page having a parameter before it. Something like this would break it:
``` bash
https://api.github.com/organizations/17532995/teams?page=2
```

Is there a reason we need to match against the &?

Reply:
ok will double check